patches/api/0291-More-World-API.patch

```diff
@@ -109,6 +109,15 @@ index e149ab9b4ccd1b98791c3ebe5e9c7eb97de853af..145780fc900531687036003d3eec2057
 +     */
 +    @NotNull
 +    Collection<Material> getInfiniburn();
++
++    /**
++     * Posts a specified game event at a location
++     *
++     * @param sourceEntity optional source entity
++     * @param gameEvent the game event to post
++     * @param position the position in the world where to post the event to listeners
++     */
++    void sendGameEvent(@Nullable Entity sourceEntity, @NotNull GameEvent gameEvent, @NotNull Vector position);
```

Comment:
`@Nullable`for Entity will not work see other comment.

Reply:
It will work if the other thing is fixed :)